ios/FluentUI/People Picker/PeoplePicker.swift

```diff
@@ -113,9 +113,9 @@ open class PeoplePicker: BadgeField {
 
     private var containingViewBoundsObservation: NSKeyValueObservation?
 
-    private let personaSuggestionsView = UIView()
+    public let personaSuggestionsView = UIView()
```

Comment:
>public [](start = 4, length = 6)

please add xcode documentation to any public member

place the cursor above the method/var, then Option+Command+/ for automatic format

Reply:
added